packages/design-system-docs/src/pages/components/DateField/DateField.example.jsx

```diff
@@ -44,6 +44,20 @@ ReactDOM.render(
       yearInvalid
     />
     <ControlledDateField />
+    <fieldset className="ds-u-margin-top--3 ds-u-padding-left--3 ds-u-padding-bottom--3 ds-u-border--2">
```

Comment:
I almost wonder if we could just not include a new example, as its currently hard to test anyway. Maybe we could add a new example in the future when we get confirmation from UKgov on how to test?

Reply:
That'a a good point. We'll wait for confirmation form UKgov on how to acceptably test this feature before adding an example.

Reply:
Just curious, how did you end up reaching out to UKgov?

Reply:
I emailed using this link https://design-system.service.gov.uk/get-in-touch/ 
and have received a reply which I'm following up on.